typings/mongoengine/queryset/queryset.pyi

```diff
@@ -75,19 +78,35 @@ class QuerySet(Generic[_T]):
     @overload
     def insert(
         self,
-        doc_or_docs: Union[_T, List[_T]],
+        doc_or_docs: _INSERT_TYPE,  # pyright: reportInvalidTypeVarUse=false
```

Comment:
what's this about?

Reply:
Pylance doesn't like us using a typevar for a function when it's only used once. It wants us to manually specify the type, but this works.

Reply:
Oh because really in that case it's equivalent to the `bound`